packages/react-composites/src/composites/common/ParticipantContainer.tsx

```diff
@@ -75,3 +72,16 @@ export const ParticipantListWithHeading = (props: {
     </Stack>
   );
 };
+
+const participantListDesktopStyle = mergeStyles({
+  maxWidth: '20rem',
+  height: '100%',
+  overflowY: 'auto',
+  overflowX: 'hidden'
+});
+const participantListMobileStyle = mergeStyles({
+  maxWidth: '100vw',
```

Comment:
don't use `vw` as that is the entire viewport width. While our mobile composite is likely to fill the screen it's not gauranteed. So we need the width of this list to fill the container it is in (100%) over being sized to the viewport